package.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Is this new file needed here?

Reply:
No, that can be ignored.

Reply:
Alright, if so, can it be removed from this PR?

Reply:
yes, for sure. Deleting it from the PR.